src/SearchUsersPage/SearchUsersPage.tsx

```diff
@@ -115,24 +123,21 @@ const SearchUsersPage: FC<SearchUsersPageProps> = ({
 					<Button
 						color="primary"
 						onClick={() => {
-							setIsLoading(true)
-							userSearchFunc(
-								context.api,
-								searchValue,
-								setResult
-							).then(() => {
-								setHasSearched(true)
-								setIsLoading(false)
-							})
+							fireUserSearch(searchValue)
 						}}
 						variant="contained"
 					>
 						Search
 					</Button>
 				</Grid>
 			</Grid>
+			{hasSearched && !isLoading && !!searchError && (
```

Comment:
Why do you have a double `!`? 2 wrongs makes a right?

Reply:
The `!!` (pronounced "double bang") is a common JS/TS technique to convert ANY expression to a strictly boolean value. More info - https://javascript.plainenglish.io/what-is-double-bang-operator-in-javascript-90fc67ead5a4

Some examples - 

![image](https://user-images.githubusercontent.com/2125913/125208662-1bb4e380-e251-11eb-8c46-5b3d5a2da487.png)